src/pages/ProductListing/ProductListing.css

```diff
@@ -0,0 +1,26 @@
+:root {
+    --bg-color: rgb(255, 255, 255);
+    --body-bgcolor: rgb(230, 230, 230);
+}
```

Comment:
You can declare these values in `index.css` and use them at every place wherever you required them instead of redeclaring them in every file.

Reply:
ok